client/unit-tests/schemas/test_label.py

```diff
@@ -51,21 +51,21 @@ def test_scored_label():
     assert l1.value == "value"
 
     # test member fn `__eq__`
-    assert (s1 == s2).get_value()  # type: ignore - resolved to 'Bool' as both sides are values
-    assert (s1 == s6).get_value()  # type: ignore - resolved to 'Bool' as both sides are values
-    assert not (s1 == s3).get_value()  # type: ignore - resolved to 'Bool' as both sides are values
-    assert not (s1 == s4).get_value()  # type: ignore - resolved to 'Bool' as both sides are values
-    assert not (s1 == s5).get_value()  # type: ignore - resolved to 'Bool' as both sides are values
+    assert s1 == s2  # type: ignore - resolved to 'Bool' as both sides are values
+    assert s1 == s6  # type: ignore - resolved to 'Bool' as both sides are values
+    assert not (s1 == s3)  # type: ignore - resolved to 'Bool' as both sides are values
+    assert not (s1 == s4)  # type: ignore - resolved to 'Bool' as both sides are values
+    assert not (s1 == s5)  # type: ignore - resolved to 'Bool' as both sides are values
```

Comment:
Can we remove all these type ignores now?

Reply:
yep! removed